### PR TITLE
Sniper specialist prepare position ability

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
## About the PR
This makes the ghillie suit have light speed to match CM13

Adds the prepare position ability

also cleans up helmet YML so the ghillie helmet can hide markings

## Media

https://github.com/user-attachments/assets/fde42af5-ff9e-4c87-b1f0-6febebaf6df3

## Requirements

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion

:cl:
- add: Added sniper specialist's prepare position ability. This will allow them to go invisible while standing still.
- tweak: Made the sniper ghillie suit have light armor speed.
- tweak: Made the sniper ghillie suit unable to be acidifed.

### CODE REVIEW

**Comment by Dutch-VanDerLinde** [comment]
READY FOR REVIEW

**Comment by Dutch-VanDerLinde** [comment]
I have no idea if this implementation is exact parity

**Comment by DrSmugleaf** [comment]
I think it's correct, I'll check after tomorrow though

File: Content.Client/_RMC14/Armor/Ghillie/GhillieSuitSystem.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+using Content.Shared._RMC14.Armor.Ghillie;
+
+namespace Content.Client._RMC14.Armor.Ghillie;
+
+public sealed class GhillieSuitSystem : SharedGhillieSuitSystem
```
Comment (DrSmugleaf) [inline Content.Client/_RMC14/Armor/Ghillie/GhillieSuitSystem.cs:5]: You don't need a separate client and server systems if they don't have any code

File: Content.Shared/_RMC14/Armor/Ghillie/SharedGhillieSuitSystem.cs [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+            return;
+
+        if (suit.Value.Comp.Enabled
+            && TryComp<EntityActiveInvisibleComponent>(user, out var invis)
+            && TryComp<RMCPassiveStealthComponent>(user, out var passive))
+        {
+            passive.ToggleTime = _timing.CurTime;
+            Dirty(user, passive);
+
+            invis.Opacity = 1;
```
Comment (DrSmugleaf) [inline Content.Shared/_RMC14/Armor/Ghillie/SharedGhillieSuitSystem.cs:242]: In 13 this makes you slightly more visible each shot

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by Dutch-VanDerLinde** [comment]
Reviews done

> Thanks for the PR!

**Review by DrSmugleaf — APPROVED** [review]

**Comment by mmails** [comment]
Note that this ability should also be given to the PMC snipers and other ERT snipers with similar loadouts. Im not sure exactly how the ability is handled, just mentioning it for future reference

**Pull request merged**